README.md

```diff
@@ -113,20 +113,22 @@ RInAppMessaging.logEvent(CustomEvent(withName: "any_event_name_here", withCustom
 
 A preference is what will allow IAM to identify users for targeting and segmentation. At the moment, IAM will take in any of the following identifiers:
 
-1.  RakutenID
+1.  RakutenID - Any value that is considered by the app as user identifier.
 2.  UserID - The ID when registering a Rakuten account. e.g. an email address
+2.  IDTrackingIdentifier - The value provided by the internal identity SDK as the "tracking identifier" value.
```

Comment:
Should this be 3? And the below 4?

Reply:
Markdown will render it in numeric order incrementing by one from the first item in the list

Reply:
Nice catch. I'll change every number to `1.`